sway/input/cursor.c

```diff
@@ -1058,12 +1058,18 @@ uint32_t get_mouse_bindcode(const char *name, char **error) {
 	}
 	const char *event = libevdev_event_code_get_name(EV_KEY, code);
 	if (!event || strncmp(event, "BTN_", strlen("BTN_")) != 0) {
+		char *_event;
+		if (!event) {
+			_event = strdup("null");
+		} else {
+			_event = strdup(event);
+		}
 		size_t len = snprintf(NULL, 0, "Event code %d (%s) is not a button",
-				code, event) + 1;
+				code, _event) + 1;
```

Comment:
Can you just use a ternary expression? `event ? event : "(null)"`

Reply:
That is way nicer, thanks! I added a commit but I cannot test compilation until #5106 is fixed.